src/main/java/io/github/pavelapis/minesweeper/Minesweeper.java

```diff
@@ -11,9 +11,29 @@
 
 public class Minesweeper extends JPanel {
 
-    int sizeX;
-    int sizeY;
-    Cell[][] field;
+    private final int sizeX;
+    private final int sizeY;
+    private Cell[][] field;
+
+    public int getSizeX() {
```

Comment:
Use lombok

Reply:
I seted with lombok necessary getters and setters